Peaky.Tests/TestClasses/GotTests.cs

```diff
@@ -23,6 +23,11 @@ public dynamic failing_test()
             throw new Exception("oops!");
         }
 
+        public dynamic retryable_test()
+        {
+            throw new SuggestRetryException("service not available, retry later");
```

Comment:
I wonder if we want some sort of helper method that reads a little more nicely. The exception name should reflect what happened (e.g. `TestTargetUnavailableException`), not what action we expect someone to take.